src/gui/guiutility.h

```diff
@@ -51,6 +51,10 @@ namespace Utility {
 
     QString socketApiSocketPath();
 
+    void setDirectoryTag(const QString &path);
```

Comment:
Please add a description.
Could you think of a more descriptive function name?

Reply:
what about `markDirectoryAsSyncRoot(..)`?

Reply:
Better :)

Reply:
done